app/(settings)/settings.js

```diff
@@ -1,108 +1,167 @@
+                        />
+                                    {/* Reset Password Modal */}
+                <StyledModal
+                    className='w-[80%] self-center z-50'
+                    isVisible={isResetModalVisible}
+                >
+                    <StyledView className='bg-offblack border-[5px] border-yellow rounded-2xl h-[60%]'>
+                        <StyledView className='flex-1 items-center h-[60%]'>
+                            <StyledText className='top-[4%] text-3xl text-offwhite'>
+                                Change Password
+                            </StyledText>
+
+                            <StyledInput
+                                className='mt-5 p-2 w-[80%] border-[1px] border-outline rounded-xl'
+                                placeholder="Current Password"
+                                secureTextEntry={true}
+                                value={currentPassword}
+                                onChangeText={setCurrentPassword}
+                            />
+                            <StyledInput
+                                className='mt-5 p-2 w-[80%] border-[1px] border-outline rounded-xl'
+                                placeholder="New Password"
+                                secureTextEntry={true}
+                                value={newPassword}
+                                onChangeText={setNewPassword}
+                            />
+                            <StyledInput
+                                className='mt-5 p-2 w-[80%] border-[1px] border-outline rounded-xl'
+                                placeholder="Confirm New Password"
+                                secureTextEntry={true}
+                                value={confirmPassword}
+                                onChangeText={setConfirmPassword}
+                            />
+
+
+                            <Button
+                                title='Confirm'
+                                btnStyles='mt-5 border-2 border-yellow'
+                                bgColor='bg-offblack'
+                                textStyles='text-yellow'
+                                width='w-[70%]'
+                                press={ResetModal}
+                            />
+                            <Button
+                                title='Cancel'
+                                btnStyles='mt-5'
+                                width='w-[70%]'
+                                press={ResetModal}
+                            />
+                        </StyledView>
+                    </StyledView>
+                </StyledModal>
+                <Button 
+                    icon='person-circle-outline'
+                    href='/mainViewLayout'
+                    btnStyles={'absolute bottom-10 left-5'}
+                    width={'w-[60px]'}
+                    height={'h-[60px]'}
+                    iconSize={40}
+                >
+                </Button>
```

Comment:
This button isn't properly placed from the bottom. To fix this, place it inside a StyledView, and add `style={{bottom: insets.bottom}}`